### DISCUSSION TITLE
Includes or Preload like Features

### DISCUSSION THREAD

**Discussion by anurag925** [aarondl/sqlboiler#1201] [Q&A]
In case of rails active record there is (Model.include(:relationship, complex_relation: :complex_relation).all) a include functionality that helps us load all the relationships so as to avoid N+1 query. can we do it in sqlboiler?... if yes how?   
i found one discussion about qm.Load() but didn't get it fully

**Comment by stephenafamo** [comment]
You can use `qm.Load()` to load relationships. There are some examples in the README.md
